BackEnd/src/business/sheets/draftParser.ts

```diff
@@ -1,11 +1,30 @@
 import { RisenSheetParser, RisenTeam } from './sheets';
-// TODO when we have reqs for draft
+import { getPlayerIdentifierFromString, hasValidOPGG } from './parserHelpers';
 export class DraftParser implements RisenSheetParser {
   public buildTeam(row: any[]): RisenTeam {
-    return null;
+    return {
+      teamName: row[0],
+      abrv: row[1],
+      win: row[2],
+      loss: row[3],
+      top: getPlayerIdentifierFromString(row[7]),
+      jungle: getPlayerIdentifierFromString(row[9]),
+      mid: getPlayerIdentifierFromString(row[11]),
+      adc: getPlayerIdentifierFromString(row[13]),
+      support: getPlayerIdentifierFromString(row[15]),
+      sub1: undefined, // No defined subs in draft league
```

Comment:
What about the rest of the leagues lol

Reply:
Premade is covered via the premadeParser, this is the draft league parser which is a diff format